traffic_ops/client/delivery_service.go

```diff
@@ -161,6 +161,30 @@ type DeliveryServiceServer struct {
 	DeliveryService string `json:"deliveryService"`
 }
 
+// DeliveryServiceSSLKeysResponse ...
+type DeliveryServiceSSLKeysResponse struct {
+	Response DeliveryServiceSSLKeys `json:"response"`
+}
+
+// DeliveryServiceSSLKeys ...
+type DeliveryServiceSSLKeys struct {
+	BusinessUnit string                            `json:"businessUnit"`
+	City         string                            `json:"city"`
+	Organization string                            `json:"organization"`
+	Hostname     string                            `json:"hostname"`
+	Country      string                            `json:"country"`
+	State        string                            `json:"state"`
+	Version      string                            `json:"version"`
+	Certificate  DeliveryServiceSSLKeysCertificate `json:"certificate"`
```

Comment:
I think we want to add Cdn and Deliveryservice fields to this struct...here is a sample output: 

```
{
  "response": {
    "country": "US",
    "cdn": "qa",
    "deliveryservice": "my-ds",
    "org": "Kabletown",
    "version": "1",
    "certificate": {
      "csr": "mycsr",
      "crt": "mycrt",
      "key": "mykey"
    },
    "hostname": "*.my-ds.qa.kabletown.net",
    "key": "my-ds",
    "state": "Colorado",
    "city": "Denver",
    "unit": "CDNENG"
  }
}

```